### ISSUE TITLE
jenkins::cli_helper and cli exec not working with key file param

### ISSUE THREAD

**Issue by athingisathing** [jenkinsci/puppet-jenkins#482]
if you have an execution path through your puppet code that  calls jenkins::cli::exec and security is enabled with this:

```
      class {'::jenkins':
        cli_ssh_keyfile => $system_user_key,
      }
```
then

this call will not use the key file.

 ```    
 jenkins::cli::exec { 'setup secure user':
        command => "create_or_update_user '${admin_user}@xxxx.com' '${admin_user}@xxxx.com' '${admin_password}' '${admin_user}: Managed by Puppet' '${system_user_pub}'",
      }
```
it ignores the key file because in the helper class which exec uses it does not use the proper key file var name
```
  if $ssh_keyfile {
    $auth_arg = "-i ${ssh_keyfile}"
  } else {
    $auth_arg = undef
  }
```
instead of $cli_ssh_keyfile  

it should use that param or check them both.

**Comment by jhoblitt** [comment]
Is this with the present master or or v1.6.1?

**Comment by br0ch0n** [comment]
FYI I just checked; it's fixed in current master but is broken in 1.6.1

**Comment by jhoblitt** [comment]
I'm going to close this since the issue is already fixed.
